dashboard/modules/version.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+# Version 0 -> 1: Added log streaming and changed behavior of job logs cli.
+CURRENT_VERSION = "1"
+
```

Comment:
Nice work adding this file! Could we also have the files in the `python/ray/dashboard/modules/job/` director and the job tests inherit from here too? Currently, they're using the version objects stored in the `python/ray/dashboard/modules/job/common.py` file.

Reply:
Ah, never mind. Just saw that you had already updated the `job` code to use the new `version.py` file. I'll resolve this.